leptonai/cli/deployment.py

```diff
@@ -519,14 +520,11 @@ def _create_workspace_token_secret_var_if_not_existing(client: APIClient):
                 Use this option to set a threshold for QPM and enable the system to scale between
                 a minimum and maximum number of replicas. For example,
                 to scale between 1 (min_replica) and 3 (max_replica) with a 2.5 QPM,
-                use: --autoscale-between 1,3,2.5
+                use: --autoscale-qpm 1,3,2.5
                 (Note: Do not include spaces around the comma.)
 
-                If the QPM is higher than the target QPM,
-                the autoscaler will scale up the replicas.
-                If the QPM is lower than the target QPM,
-                the autoscaler will scale down the replicas.
-                The threshold value should be between positive number.
+                This sets up autoscaling based on queries per minute, scaling between 1 and 3 replicas when QPM 
```

Comment:
this line may be too long but I don't know how you break lines

Reply:
refined